### ISSUE TITLE
Form validation: 

### ISSUE THREAD

**Issue by jdm000** [wet-boew/wet-boew#9084]
For customized pattern validation, the documentation here leaves out the fact that the "pattern" attribute is stripped out by WET (at least for the version I'm running).  Maybe it should mention in that if "pattern" isn't being recognized, to use the attribute "data-rule-pattern" in its place.

**Comment by RobJohnston** [comment]
Yes, something should be mentioned, as this "pattern" thing keeps coming up.  I thought about adding an example a long time ago, but it wasn't workable:  https://github.com/wet-boew/wet-boew/issues/7226#issuecomment-160997645

**Comment by RobJohnston** [comment]
Correction... I had forgotten that the "pattern" attribute has been added recently.  See https://github.com/wet-boew/wet-boew/pull/8983.

**Comment by jdm000** [comment]
As someone attempting to use the WET provided validation for the first time, I wasn’t aware of the history of the validation.  I didn’t even know it does or (used to) leverage jQuery for the validation, beyond just DOM manipulation.  I also don’t have full control over what version of WET my agency is using.

So I had to do some considerable digging into the WET code to figure out why the pattern attribute wasn’t being recognized with my version of WET, despite what the documentation here indicated.  I also had to do digging to figure out what would work in its place.

A simple sentence or two in the documentation is all that’s needed here, for those not running the most up to date version.

**Comment by duboisp** [comment]
@jdm000 we added the support for the pattern attribute. You can find a working example in the [form validation plugin](https://wet-boew.github.io/wet-boew/demos/formvalid/formvalid-en.html).

**Comment by duboisp** [comment]
@jdm000 feel free to submit a PR with those 2 sentence that will provide more clarity. 

You can edit the following files:
* https://github.com/wet-boew/wet-boew/blob/master/src/plugins/formvalid/formvalid-en.hbs#L452
* https://github.com/wet-boew/wet-boew/blob/master/src/plugins/formvalid/formvalid-fr.hbs#L452
